src/vs/base/browser/ui/button/button.css

```diff
@@ -46,6 +46,11 @@
 	padding: 4px 0;
 }
 
+.monaco-button-dropdown .monaco-button-dropdown-separator.disabled {
+	opacity: 0.4 !important;
```

Comment:
Any chance we can get this without `!important`?

Reply:
Same comment for all other `!important` uses in this file. The issue with `!important` clauses is that they are sticky. Things that want to build on top of them will require `!important` as well. And then, once everything is `!important`, we no longer have a way to overwrite things, ie there's no `!superimportant`.

Reply:
Removed the `!important`.